operator/v1/types_ingress.go

```diff
@@ -344,6 +344,47 @@ type IngressControllerSpec struct {
 	// +kubebuilder:default:="Immediate"
 	// +default="Immediate"
 	IdleConnectionTerminationPolicy IngressControllerConnectionTerminationPolicy `json:"idleConnectionTerminationPolicy,omitempty"`
+
+	// closedClientConnectionPolicy controls how the IngressController
```

Comment:
Would `clientDisconnectPolicy` be a better name for this?

Reply:
"clientDisconnect" is a good name, concise and descriptive, it didn't come up to me. My choice for "ClosedConnection" was driven by the upstream wording, HAProxy always uses "closed connection" or "half closed connection" when referring to the config option we are introducing. Also, I tend to interpret "Disconnect" as a fully closed TCP connection (both sides) while here we are dealing with a connection closed from the client side.